core/sql/generator/GenRelScan.cpp

```diff
@@ -2183,6 +2182,17 @@ short HbaseAccess::codeGen(Generator * generator)
   if (getTableDesc()->getNATable()->hasAddedColumn())
     hasAddedColumns = TRUE;
 
+  NABoolean alignedFormatTable = 
+    getTableDesc()->getNATable()->isSQLMXAlignedTable();
```

Comment:
Will this line conflict with Selva's change? (See https://github.com/apache/incubator-trafodion/pull/42). I'm wondering about ths isSQLMXAlignedTable() call in particular.